content/zh/news/releases/1.20.x/announcing-1.20.5/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 发布 Istio 1.20.5
+linktitle: 1.20.5
+subtitle: 补丁发布
+description: Istio 1.20.5 补丁发布。
+publishdate: 2024-04-08
+release: 1.20.5
+---
+
+本次发布实现了 4 月 8 日公布的安全更新 [`ISTIO-SECURITY-2024-002`](/zh/news/security/istio-security-2024-002)
+并修复了一些错误，提高了稳健性。
+
+本发布说明描述了 Istio 1.20.4 和 Istio 1.20.5 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了当 `VirtualService` 包含不同情况的重复主机将导致路由被 Envoy 拒绝的错误。
+  ([Issue #49368](https://github.com/istio/istio/issues/49368))
```

Comment:
Wait this [#49368](https://github.com/istio/istio/issues/49368) is not a issue, but an automator job...

Reply:
Let me check it.

Reply:
Yes, looks like the original English content is incorrect as well.

Reply:
Let me correct.

Reply:
Done in Chinese part.